doc/themes/nilearn/layout.html

```diff
@@ -261,6 +261,8 @@ <h2>Machine learning for Neuro-Imaging in Python</h2>
 
 <h4> News </h4>
   <ul>
+      <li><p><strong>February 2020</strong>: Nilearn 0.6.2 released
+      </p></li>
```

Comment:
Can you also remove some of the old ones: let's keep that list no longer than 4 items.

Thanks!